eureka2-core/src/main/java/com/netflix/eureka2/interests/ChangeNotifications.java

```diff
@@ -55,9 +67,20 @@ public Boolean call(ChangeNotification<?> notification) {
                 }
             };
 
+    private static final Comparator<InstanceInfo> INSTANCE_INFO_IDENTITY_COMPARATOR = new Comparator<InstanceInfo>() {
+        @Override
+        public int compare(InstanceInfo o1, InstanceInfo o2) {
+            return o1.getId().compareTo(o2.getId());
+        }
+    };
+
     private ChangeNotifications() {
     }
 
+    public static Comparator<InstanceInfo> instanceInfoIdentityComparator() {
```

Comment:
To align it with rest of the methods in this class. They are all factories for functions.